frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -405,6 +415,12 @@ class DashboardHeader extends Component {
     return buttons;
   }
 
+  saveAsImage = async () => {
+    const { dashboard } = this.props;
+    const cardNodeSelector = `[data-testid="Dashboard-Cards-Container"]`;
```

Comment:
It would be better to use `data-testid` only in tests, can we use a different selector?

Reply:
I'm not totally sure, but we could assign a regular HTML id rather than a test-id if not.